integreat_cms/cms/forms/custom_content_model_form.py

```diff
@@ -172,7 +275,12 @@ def clean_content(self) -> str:
                         "Updated link url from %s to %s", href, translated_url
                     )
 
-        # Scan for media files in content and replace alt texts
+    def process_images(self, content: HtmlElement) -> None:
+        """
+        This function processes images by scanning for media files and replacing alt texts.
```

Comment:
To be honest I don't yet fully understand what we need this function for. Therefore the comment is not yet containing the explanation of why we do what we do here

Reply:
I think this is useful when someone links to some image that is stored in the media library. Then we want to have an alt text for that image for better accessibility and since we store alt texts in the media library, we can just use those. In most cases, tinymce should already add the correct alt texts, but this function should take care for the remaining cases (and it also fixes outdated alt texts when the alt texts in the media library were changed)